pages/index.js

```diff
@@ -1,11 +1,22 @@
 import ItemList from '../components/ItemList';
 import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import { dummyitems } from '../lib/data';
 
 export default function Home() {
+  const [items, setItems] = useState([...dummyitems]);
+  function handleAddItem(newItem) {
```

Comment:
Try and name your "brokkolis" the same when possible. You named the argument you passed in AddForm "data", I suggest you name both newItem :)